modules/detect_target/detect_target_contour.py

```diff
@@ -0,0 +1,137 @@
+"""
+Detects objects using the provided model.
+"""
+
+import time
+
+import cv2
+import numpy as np
+
+from . import base_detect_target
+from .. import image_and_time
+from .. import detections_and_time
+from ..common.modules.logger import logger
+
+
+MIN_CONTOUR_AREA = 100
+MAX_CIRCULARITY = 1.3
+MIN_CIRCULARITY = 0.7
+UPPER_BLUE = np.array([130, 255, 255])
```

Comment:
Add empty line above this line. This makes it more readable (above is geometry, middle is colour, bottom is values returned).